### ISSUE TITLE
Monitoring non-output values

### ISSUE THREAD

**Issue by maxbry** [fchollet/keras#4644]
My network has two different outputs, let's call them `A` and `B`. While training, I have 10 monitoring values: loss, loss_A, loss_B, binary_accuracy_A, binary_accuracy_B, val_loss, val_loss_A, val_loss_B, val_binary_accuracy_A, val_binary_accuravy_B.
My problem is that I am not interested in all of these outputs. I would like to determine which are the ones I see and I want to create my own monitor value.
My own monitor value should be something like the percantage `A > B`.

Is this possible?

**Comment by bkj** [comment]
I believe you could do this by writing a custom callback: https://keras.io/callbacks/
